src/main/java/org/myrobotlab/service/meta/abstracts/AbstractSpeechSynthesisMeta.java

```diff
@@ -0,0 +1,26 @@
+package org.myrobotlab.service.meta.abstracts;
+
+import org.myrobotlab.logging.LoggerFactory;
+import org.slf4j.Logger;
+
+public class AbstractSpeechSynthesisMeta extends MetaData {
+  private static final long serialVersionUID = 1L;
+  public final static Logger log = LoggerFactory.getLogger(AbstractSpeechSynthesisMeta.class);
+
+  /**
+   * This class is contains all the meta data details of a service. It's peers,
+   * dependencies, and all other meta data related to the service.
+   * 
+   */
+  public AbstractSpeechSynthesisMeta() {
+
+    addPeer("audioFile", "AudioFile", "audioFile");
+    addCategory("speech");
+
+    addDependency("org.myrobotlab.audio", "voice-effects", "1.0", "zip");
+    addDependency("javazoom", "jlayer", "1.0.1");
```

Comment:
This class is useless.  None of the subclasses call  super() as far as i can tell. . and even if they did.. why would every speech synt force jlayer and mp3spi as a dependency.  Either make this class useful or remove it.

Reply:
The dependencies can go but the peer is critical, so is this class.
Calling super? Search for derived classes and you'll find all the meta speech synthesis that cache their mp3 files.
If you removed it you'd have 6 speech synthesis services without cache.

Reply:
This seems to have been my confusion.  I didn't realize that the constructor of the parent class was called implicitly,  I was always taught that you had to explicitly call super() to call the parent method... I assumed that applied to consturctros also.  (that being said.. the mp3 stuff is really part of audioFile ..so.. cool.)